src/FunctionApps/python/IntakePipeline/.gitignore

```diff
@@ -0,0 +1,48 @@
+bin
```

Comment:
Is this any different than the one in the root?

Reply:
Nope. Removed this instance of it.